CONTRIBUTING.md

```diff
@@ -121,3 +124,9 @@ lxcore_kernel.etl
 lxcore_service.etl
 lxcore_user.etl
 ```
+
+#### Submitting logs
```

Comment:
The ETL files do not contain sensitive information.

Reply:
Cool. I'll revise. (This was part of the uncertainty I had, hence the reason I opened #3137.)